.gitmodules

```diff
@@ -1,4 +1,3 @@
 [submodule "xgboost-sys/xgboost"]
 	path = xgboost-sys/xgboost
-	url = https://github.com/davechallis/xgboost
-	branch = master
+	url = https://github.com/dmlc/xgboost
```

Comment:
iirc, this was originally pointed at a fork to deal with some build issues with the original library involving unused packages. Hopefully it's fixed now though (I'm hazy on the details, but was something to do with the python bindings perhaps...).